arkouda/util.py

```diff
@@ -292,10 +292,13 @@ def attach_all(names: list):
     return {n: attach(n) for n in names}
 
 
-def sparse_sum_help(idx1, idx2, val1, val2):
+def sparse_sum_help(idx1, idx2, val1, val2, merge=True):
```

Comment:
Is there a way to unit test this?

Reply:
Good question! there is... with a catch lol. A slightly modified version of the quick and dirty benchmark I had ben run would work to verify we are matching the sum aggregation. The catch is that our CI doesn't run multi-locale and that's the only case where this could run into trouble, so it wouldn't be super useful for us

But when this gets pulled onto user machines if they run `make test` with a multi-locale build, it would give us some info if that test failed. So it is worth adding

Reply:
Added!

Reply:
Thanks!